components/GetMetar.js

```diff
@@ -0,0 +1,49 @@
+const noflo = require('noflo');
+const metar = require('metar');
+// Polyfill for fetch()
+require('isomorphic-fetch');
+
+exports.getComponent = function () {
```

Comment:
`GetMetar` is a pretty cryptic component name. Better something like `GetWeatherMetar` or `GetMetarWeather`

Reply:
Done